docs/developer-docs/multi-chain/ethereum/evm-rpc/samples.mdx

```diff
@@ -0,0 +1,23 @@
+---
+keywords: [advanced, ethereum, evm rpc, evm, tutorial]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# EVM RPC samples
+
+<MarkdownChipRow labels={["Advanced", "Ethereum", "Tutorial"]} />
+
+Here are some example projects using the EVM RPC canister:
+
+- [Vite + React + Motoko  + EVM RPC](https://github.com/rvanasa/vite-react-motoko/tree/evm-rpc)
+
+- [Vite + React + Rust + EVM RPC](https://github.com/fxgst/evm-rpc-rust/tree/main)
+
+- [ICP ETH Coprocessor](https://github.com/domwoe/icp-eth-coprocessor/tree/main)
+
+- [Multi-chain voting](https://github.com/bjoernek/multi_chain_voting/tree/main)
+
+- [ICP + ETH starter (Rust)](https://github.com/benjizhai/ic-eth-starter-rust/tree/main)
```

Comment:
i think this is not using the EVM rpc canister that we have today, is a bit outdated